image/db/rhel/konflux.Dockerfile

```
@@ -1,14 +1,11 @@
-FROM registry.redhat.io/rhel8/postgresql-12:latest
+FROM registry.redhat.io/rhel8/postgresql-12:latest as scanner-db-common
```

[minor cosmetics] Even though there's no restriction, I think, it's common to write `as` in all-caps to match style of `FROM`.

```suggestion
FROM registry.redhat.io/rhel8/postgresql-12:latest AS scanner-db-common
```